tsconfig.json

```diff
@@ -6,6 +6,15 @@
     "module": "ESNext",
     "skipLibCheck": true,
 
+    /* Aliases */
+    "baseUrl": "./src",
+    "paths": {
+      "@components/*": ["components/*"],
```

Comment:
So usually when we work with aliases we use names without `@` since `@` is often used for namespaces and it makes confusing the two.